manageiq-loggers.gemspec

```diff
@@ -23,6 +23,7 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency "activesupport",     ">= 5.0"
   spec.add_runtime_dependency "manageiq-password", "~> 0.1"
+  spec.add_runtime_dependency "systemd-journal",   "~> 1.4.0"
```

Comment:
Let's make this optional

Reply:
Done

Reply:
@Fryguy not sure if you know of a better way to do this, but I dropped the gem from this gemspec so that the "optional" part can be controlled by the app that this is bundled into (aka ManageIQ core), and removed the require of `manageiq/loggers/journald` from `manageiq/loggers` so it isn't automatically required.

Reply:
Yeah that's the only way to do it...optional deps have been a rubygems RFE for ages, but it's never been seen as needed.

Reply:
:+1: done over in https://github.com/ManageIQ/manageiq/pull/18648/files#diff-8b7db4d5cc4b8f6dc8feb7030baa2478R134